client/boot/common.js

```diff
@@ -225,9 +225,7 @@ export const configureReduxStore = ( currentUser, reduxStore ) => {
 	}
 };
 
-export const setupMiddlewares = ( currentUser, reduxStore ) => {
-	debug( 'Executing Calypso setup middlewares.' );
-
+export const setupBasicMiddlewares = ( currentUser, reduxStore ) => {
```

Comment:
I'm certain that we won't be needing some of those middlewares for the Jetpack Cloud. So, one alternative way to go here would be to export all of them in `boot/common`, but then import only the ones we need for the Jetpack Cloud boot.